helpers/manifest-parser.js

```diff
@@ -141,8 +141,13 @@ function parseIcon(raw) {
   let sizes = parseString(raw.sizes);
   if (sizes.value !== undefined) {
     let set = new Set();
-    sizes.value.split(/\s/).forEach(size => set.add(size.toLowerCase()));
-
+    sizes.value.split(/\s/).forEach(size => {
+      const pair = size.split(/x/i);
+      if (pair[0] !== pair[1]) {
```

Comment:
right, but this is being generated in a gather (manifest.js), so it needs a particular audit to decided to display it. Currently this will just be dropped.